internal/provider/requestablepermission_resource_test.go

```diff
@@ -33,4 +35,87 @@ func TestAccPermissionResource(t *testing.T) {
 			},
 		},
 	})
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		ErrorCheck: func(err error) error {
+			if err == nil {
+				return errors.New("expected error but got none")
+			}
+
+			// some simple example with string matching
+			if strings.Contains(err.Error(), "Invalid request approval config") {
+				return nil
+			}
+
+			// return original error if no match
+			return err
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: providerConfig() + fmt.Sprintf(`
+				resource "lumos_requestable_permission" "test" {
+					app_id      = "%s"
+					label       = "Permission Label"
+					request_config = {
+						appstore_visibility = "HIDDEN"
+						request_approval_config = {
+						  approvers = {
+							users  = []
+							groups = []
+						  },
+						  manager_approval = "NONE"
+						}
+					  }
+				}`, os.Getenv("APP_ID")),
+			},
```

Comment:
Is it necessary to have this first `Step`? I would think the same error would be thrown if an approver is included without the override flag being set during creation, not just during an update

Reply:
good question let me try